pkg/storage/system.go

```diff
@@ -0,0 +1,64 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// FetchSystemByLCID uses the Lifecycle ID as the unique identifier for a given System
+func (s *Store) FetchSystemByLCID(ctx context.Context, lcid string) (*models.System, error) {
+	// TODO: this code would emulate the behavior outlined in `ListSystems(...)`
+	return nil, fmt.Errorf("not yet implemented")
+}
+
+// ListSystems retrieves a collection of Systems, which are a subset of all SystemIntakes that
+// have been "decided" and issued an LCID.
+func (s *Store) ListSystems(ctx context.Context) ([]*models.System, error) {
```

Comment:
This func presents two options for implementation details, each "hidden" behind a nonsense placeholder conditional (`sqlComplete`)